pages/index.tsx

```diff
@@ -1,65 +1,5 @@
-import Head from 'next/head'
-import Image from 'next/image'
-
-export default function Home() {
+export default function DashboardPage() {
```

Comment:
why is this changed? this should be coming from main now, so just `git pull origin main`. if you did that lmk there may be some weirdness with git

Reply:
It was probably me not doing that on accident and doing something funky instead. Should be fixed now.